_data/entertainment.yml

```diff
@@ -122,3 +112,13 @@ websites:
       twitter: vimeo
       img: vimeo.png
       tfa: No
+
+    - name: YouTube
+      url: https://www.youtube.com/
+      img: youtube.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      phone: Yes
+      hardware: Yes
+      doc: http://www.google.com/intl/en-US/landing/2step/features.html
```

Comment:
As per our `.editorconfig`, it contains `insert_final_newline = true`.
Please respect that.